cmd/vclusterctl/cmd/create.go

```
@@ -223,6 +224,14 @@ func (cmd *CreateCmd) Run(args []string) error {
 		}
 	}
 
+	workDir, err := os.Getwd()
+	if err != nil {
+		log.GetInstance().Fatal(fmt.Errorf("unable to get current work directory: %v", err))
```

Please return an error here instead of killing the program directly